src/shogun/statistical_testing/internals/mmd/CrossValidationMMD.h

```diff
@@ -195,16 +195,11 @@ struct CrossValidationMMD : PermutationMMD
 		SGVector<int64_t> dummy_labels_x(m_n_x);
 		SGVector<int64_t> dummy_labels_y(m_n_y);
 
```

Comment:
@lambday i'm pretty sure that if i would do `prng(); prng();`
this patch would pass